server/src/dtos/person.dto.ts

```diff
@@ -63,6 +63,19 @@ export class MergePersonDto {
 export class PersonSearchDto {
   @ValidateBoolean({ optional: true })
   withHidden?: boolean;
+
+  @ApiProperty({ description: 'Page number for pagination', default: 1, required: false })
+  @IsInt()
+  @Min(1)
+  @Type(() => Number)
+  page: number = 1;
+
+  @ApiProperty({ description: 'Number of items per page', default: 500, required: false })
+  @IsInt()
+  @Min(1)
+  @Max(1000)
+  @Type(() => Number)
```

Comment:
I don't think you need the type annotation, do you?

Reply:
The conversion is necessary because implicit conversions are disabled.  Pagination is optional with the same defaults as before, so there shouldn't be any breaking changes.

Reply:
Ah ok cool!

Reply:
We only do because it is a query param so it would be a string otherwise.